packages/@webex/common/src/browser-detection.js

```diff
@@ -2,6 +2,19 @@ import bowser from 'bowser';
 import {memoize} from 'lodash';
 import window from 'global/window';
 
+export const getBrowserSerial = () => {
+  let browserData;
+  try {
+    browserData = window?.navigator?.userAgent
+      ? bowser.getParser(window.navigator.userAgent)
+      : {error: 'unable to access window.navigator.userAgent'};
```

Comment:
helping suggestion:

when testing, this error message could be a constant to help with checking its *set* in the branch case.